src/account-info/AccountDetailsScreen.js

```diff
@@ -99,15 +99,8 @@ export default connect<SelectorProps, _, _>((state, props) => ({
     const { userId } = props.navigation.state.params;
     return userId === undefined ? userId : getAllUsersById(state).get(userId);
   })(),
-  isActive: (() => {
-    const { userId } = props.navigation.state.params;
-    if (userId === undefined) {
-      return false;
-    }
-    const user = getAllUsersById(state).get(userId);
-    if (!user) {
-      return false;
-    }
-    return getUserIsActive(state, user.email);
-  })(),
+  isActive:
+    props.navigation.state.params.userId === undefined
+      ? false
+      : getUserIsActive(state, props.navigation.state.params.userId),
```

Comment:
This is essentially the same computational structure as for `user`, above; the two should probably be structured similarly. (And, if it's simple enough to do so, partially coalesced.)